src/mpi/init/init_thread_cs.c

```diff
@@ -8,14 +8,40 @@
 #include "mpidimpl.h"
 #include "mpi_init.h"
 
+/* Unconditioned global CS used in MPI_Init/Finalize (or other non-performance
+ * critical sections)
+ *
+ * Idealy, we should have these in mpir_thread.h so other non-performance-
+ * critical paths may use these macros to stablize from the complicated and
+ * evolving threading experiments. However, its dependency on MPIDU_macros
+ * makes it complicated.
+ */
+#if defined(MPICH_IS_THREADED)
+#define MPIR_THREAD_CS_ENTER(mutex) \
+    do { \
+        int err_ = 0; \
+        MPIDU_Thread_mutex_lock(&mutex, &err_, MPL_THREAD_PRIO_HIGH); \
```

Comment:
I don't know the naming convention in MPICH, but I think it should use `MPID_Thread_mutex_lock` instead of `MPIDU_Thread_mutex_lock` (as `MPID_Thread_mutex_create` is used in the same function).

Reply:
Yes, you are right. My negligence during copy-pasting.

Reply:
Corrected.